ExceptionCatcher.application=Programma:
ExceptionCatcher.causedBy=Veroorzaakt door:
ExceptionCatcher.exceptionMessage=Fout beschrijving:
ExceptionCatcher.loggedAction=De fout is weg geschreven.
ExceptionCatcher.message=Een onverwachte fout is opgetreden:
ExceptionCatcher.nullPointerException=NullPointerException
ExceptionCatcher.thread=Taak:
ExceptionCatcher.title=Een onverwachte fout
